spec/requests/datacite_dois/datacite_dois_spec.rb

```diff
@@ -1648,4 +1648,66 @@ def clear_doi_index
       expect(json.dig("meta", "licensesWithMissing", 1, "id")).to eq("__missing__")
     end
   end
+
+  describe "GET /dois search with author values", prefix_pool_size: 1 do
+    let!(:dois) { create_list(:doi, 10, client: client, aasm_state: "findable") }
+    let!(:dois_with_creators) do
+      create_list(:doi, 3, client: client, aasm_state: "findable",
+        creators: [
+          {
+            "contributorType" => "Editor",
+            "name" => "TEST CREATOR 1",
+            "nameIdentifiers" => [
+              {
+                "nameIdentifier" => "https://orcid.org/0000-0003-3484-6874",
+                "nameIdentifierScheme" => "ORCID",
+                "schemeUri" => "https://orcid.org",
+              },
+            ],
+          },
+          {
+            "contributorType" => "Editor",
```

Comment:
`contributorType` isn't supported for creators, so I think you can remove those here even though it shouldn't really matter.

Reply:
removed